interview-questions.md

```diff
@@ -4,65 +4,112 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is a block of code that is used to perform a specific task.
```

Comment:
👍 Totally! I'd maybe mention that because its a function, its more valuable as its reusable!